### PR TITLE
Documentation pipeline fix

### PR DESCRIPTION
**Pull request by drakon660**
only for tests

### CODE REVIEW

**Comment by razvangoga** [comment]
@drakon660 does this need to be an active PR ? it's spamming workflow runs for CI and email notifications.

**Comment by razvangoga** [comment]
@drakon660 it could be made draft

**Comment by drakon660** [comment]
@razvangoga please check if you see this https://tye-v2.github.io/tye2/ correctly and approve this PR.

File: .github/workflows/deploy-gh-pages.yml [hunk 1]
```diff
@@ -4,7 +4,6 @@ name: Deploy VitePress site to Pages
 on:
   # Runs on pushes targeting the default branch
   push:
-    branches: ["main"]
```
Comment (razvangoga) [inline .github/workflows/deploy-gh-pages.yml:7]: let's restrict the workflow to only starting if the changes are in the /docs folder : https://docs.github.com/en/actions/using-workflows/triggering-a-workflow#example-including-paths

File: docs/.vitepress/config.mts [hunk 2]
```diff
@@ -2,7 +2,8 @@ import { defineConfig } from 'vitepress'
 
 // https://vitepress.dev/reference/site-config
 export default defineConfig({
-  base: '/tye-v2.github.io/',
+  ignoreDeadLinks: true,
+  base: '/tye2/',
   title: "Tye 2",
   description: "Tye",
```
Comment (razvangoga) [inline docs/.vitepress/config.mts:8]: tye 2 :)

**Review by razvangoga — CHANGES_REQUESTED** [review]

**Comment by razvangoga** [comment]
> @razvangoga please check if you see this https://tye-v2.github.io/tye2/ correctly and approve this PR.

it looks very good! great job!

**Comment by razvangoga** [comment]
@drakon660 i guess that with having this type of docs (github pages), i could disable the repo's wiki so it no longer shows in the top menu bar? right?

File: .github/workflows/deploy-gh-pages.yml (hunk 1 above)
Comment (drakon660) [reply .github/workflows/deploy-gh-pages.yml:7]: done

**Review by razvangoga — APPROVED** [review]

**Pull request merged**